README.md

```diff
@@ -31,26 +32,35 @@ Clone the repository
     .venv\Scripts\activate.bat
 
 ### Install dependencies
+
 From the top-level directory enter the command to install pip and the dependencies of the project
 
     python3 -m pip install --upgrade pip && pip install -r requirements.txt
 
 ## How to use
+
 Enter the virtual environment as described above, then:
 
     flask run
 
+    Note: This service is an internal service so it doesn't have the frontend.
+
+### Set-up an API KEY that requires to connect with the govuk-notify-service
+
+    Click on the link & follow the instructions to set-up an API KEY.
+    https://docs.notifications.service.gov.uk/python.html#api-keys
+
```

Comment:
Set it as an environment variable e.g.
`export TEST_API_KEY=some-value`

Reply:
Updatded!